src/main/java/io/cdap/plugin/google/drive/source/GoogleDriveSourceConfig.java

```diff
@@ -0,0 +1,295 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.google.drive.source;
+
+import com.google.common.base.Strings;
+import io.cdap.cdap.api.annotation.Description;
+import io.cdap.cdap.api.annotation.Macro;
+import io.cdap.cdap.api.annotation.Name;
+import io.cdap.cdap.api.data.schema.Schema;
+import io.cdap.cdap.etl.api.FailureCollector;
+import io.cdap.plugin.google.drive.common.GoogleDriveBaseConfig;
+import io.cdap.plugin.google.drive.common.GoogleDriveClient;
+import io.cdap.plugin.google.drive.common.exceptions.InvalidPropertyTypeException;
+import io.cdap.plugin.google.drive.source.utils.BodyFormat;
+import io.cdap.plugin.google.drive.source.utils.ExportedType;
+import io.cdap.plugin.google.drive.source.utils.ModifiedDateRangeType;
+import io.cdap.plugin.google.drive.source.utils.ModifiedDateRangeUtils;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.annotation.Nullable;
+
+/**
+ * Configurations for Google Drive Batch Source plugin.
+ */
+public class GoogleDriveSourceConfig extends GoogleDriveBaseConfig {
```

Comment:
update descriptions according to the updates in md doc.

Reply:
done